core_lib/src/interface/editor.cpp

```diff
@@ -73,12 +65,15 @@ Editor::Editor(QObject* parent) : QObject(parent)
     clipboardBitmapOk = false;
     clipboardVectorOk = false;
     clipboardSoundClipOk = false;
+
+    mActiveFramePool.reset(new ActiveFramePool(200));
```

Comment:
Not sure if I like the constant size here. People with better/worse machines may want this adjusted, although at the same time I don't want to add unnecessary complexity to the preferences UI so maybe this is fine. How did you arrive at the number 200?

Reply:
An HD image take about 7.91MB (1920x1080x32bit = 7.91MB) 
200 frames x 7.91MB = 1582MB ~= 1.5G

In 32bit Windows, one process can manage up to 2GB memory, 500MB reserved by kernel space, rest part is about 1.5GB in user space.

It's a conservative approach to this issue. A better idea is to detect the os architecture and memory size of user's PC, for example, the user has 16GB ram and 64bit OS, and then set a higher value. 

Since there is no way to know the actual memory size in Qt, we have to do it in a native way for each platform.